api/src/main/java/org/openmrs/scheduler/timer/TimerSchedulerServiceImpl.java

```diff
@@ -542,5 +542,15 @@ public void scheduleIfNotRunning(TaskDefinition taskDef) {
 			}
 		}
 	}
+
+	/**
+	 * Get the task with the given uuid.
```

Comment:
Can we avoid duplicating this?

Reply:
sure, removed it to, requesting for review